commcare_connect/opportunity/forms.py

```diff
@@ -43,7 +42,6 @@ def __init__(self, *args, **kwargs):
         self.helper.layout = Layout(
             Row(Field("name")),
             Row(Field("active", css_class="form-check-input", wrapper_class="form-check form-switch")),
-            Row(Field("auto_approve_visits", css_class="form-check-input", wrapper_class="form-check form-switch")),
```

Comment:
Don't we need to remove both of these toggles based on the description? If the idea is to make it impossible to reject a payment when the forms have no flags, we would need to auto approve both the visit and the payment

Reply:
This is from Rama
> [3:30 PM](https://dimagi.slack.com/archives/D057T7G55DJ/p1717063216534179)
Rama Siva
:palm_tree: For payment approval, we should leave the toggle button there and turn it on by default